source/arm11/open_agb_firm.c

```diff
@@ -110,23 +111,21 @@ static OafConfig g_oafConfig =
 
 	// [game]
 	0,     // saveSlot
+	-1,    // saveType
```

Comment:
The type is unsigned. It would make more sense to use 0xFF.

Reply:
My reasoning for it being -1 instead of 0xFF is (imo) -1 is easier for someone to remember than 0xFF/255, especially for someone who does not understand the significance of 0xFF/255. Additionally, I would be more likely to associate a negative number with disabled more than a positive number (0 would be an even better option, but because it is in use we need to use something else)

Reply:
It still doesn't feel right to me. While i get what you mean the casts make this a bit ugly. Plus in this line -1 implies this field is of some signed type when it's not. I usually explain in a comment what values like 0xFF mean.